coaster-nn/src/plugin.rs

```diff
@@ -147,6 +150,15 @@ pub trait ConvolutionConfig<F> {
     }
 }
 
+/// Provides Rnn Config functionality.
+///
+/// Needs to be implemented for Operation specific configurations.
+pub trait RnnConfig<F> {
+    /// Returns the largest workspace size in bytes needed
+    /// for any of the convolution operations.
+    fn workspace_size(&self) -> usize {0}
```

Comment:
Is `0` every a valid response with the mode we use? Wouldn't it be better to require the implementor to get it right and creating awareness?

Reply:
Added in more docs for this, but basically 0 is for the native where there isn't really a 'workspace' being generated, but there will be a need for GPUs to know how much memory to allocate.